packages/create-neutrino-project/commands/init/templates/react/src/App.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './App.css';
+
+const App = () => (
```

Comment:
This may be controversial, but let's use a component here.

```js
export default class App extends Component {
  render() {
    return (
      <div className="App">
        <h1>{/* ••••••• insert the directory name here we got from the command line •••••••••/*}</h1>
      </div>
    );
  }
}
```